src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,13 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.5.1
+6 March, 2023
+
+### Bugfixes
+
+- Fixed an issue that prevented users from authenticating against the registry when using a custom secret (`.Values.registry.ingress.tlsSecret`).
```

Comment:
Can we call this a "custom TLS secret" or "separate certificate for the registry" so it's more obvious what the use case is?